source/chapters/configuration.rst

```diff
@@ -160,31 +160,37 @@ computer. **Therefore it is important to take care to choose the best Audio API
 available to you.** Refer to the following table of Audio APIs to see what the
 best choice is for your operating system.
 
-+----------------------------------------+--------------+
-| OS / Audio API                         | Quality      |
-+========================================+==============+
-| Windows / WMME                         | Poor         |
-+----------------------------------------+--------------+
-| Windows / DirectSound                  | Poor         |
-+----------------------------------------+--------------+
-| Windows / WASAPI                       | Good         |
-+----------------------------------------+--------------+
-| Windows / ASIO                         | Good         |
-+----------------------------------------+--------------+
-| Windows / WDDKMS                       | Good         |
-+----------------------------------------+--------------+
-| Mac OS X / CoreAudio                   | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / OSS                        | OK           |
-+----------------------------------------+--------------+
-| GNU Linux / ALSA                       | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / JACK (Advanced)            | Good         |
-+----------------------------------------+--------------+
-
-On Windows, if an ASIO driver is not available for your operating system, you
-can try installing `ASIO4ALL <http://asio4all.com>`_, a low-latency audio driver
-for WDM audio devices.
++-----------------------------+---------+
+| OS / Audio API              | Quality |
++=============================+=========+
+| Windows / MME               | Poor    |
++-----------------------------+---------+
+| Windows / DirectSound       | Poor    |
++-----------------------------+---------+
+| Windows / WASAPI            | OK      |
++-----------------------------+---------+
+| Windows / ASIO              | Good    |
++-----------------------------+---------+
+| Windows / WDM-KS            | Good    |
++-----------------------------+---------+
+| Mac OS X / CoreAudio        | Good    |
++-----------------------------+---------+
+| GNU Linux / OSS             | OK      |
++-----------------------------+---------+
+| GNU Linux / ALSA            | Good    |
++-----------------------------+---------+
+| GNU Linux / JACK (Advanced) | Good    |
++-----------------------------+---------+
+
+On Windows, if there is a native ASIO driver available for your soundcard, it
+might be the best choice to achieve a low latency, since it usually bypasses the
+windows kernel.
+
+For an ASIO Setup, every soundcards need to provide its own ASIO driver.
+If this is not the case, just use the WDM-KS API. To solve specific problems,
+you may consider to use `ASIO4ALL <http://asio4all.com>`_, a thin wrapper
```

Comment:
IMHO It is not recommended at all. The only reason I can think of is that you want to achieve a very low latency for your master sound, by using a "native" ASIO driver and you need a second sound card that just plays to your headphones. But this is not recommend for the average use since you will have an echo effect due to the latency difference and the two sound-card syncing. The Headphone monitor feature is broken.